my-app/src/data/beachspeech/playerData.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
so for this file, we should store the names of the players only in the backend then use the frontend to make calls to the backend. So we shouldn't get the names of our players from this doc, but u can correct me if I'm wrong but this looks like the position of each player in the lobby so can we modify this doc to:
- remove "playerName"
- change the filename to be more specific of what we want something about position
- wherever this data, create separate fake data for the names so it will mimic retrieving the names from backend

Also what is "id" used for?

Reply:
I called it playerData to just have something to reference to until we get the backend working because the place holders wouldn't show up without it. But I can change it to hard code it for now without the use of playerData.

I just wanted some randomized positioning for the islands so it wasn't just left, right and/or center. 

The id is used for "key" values in React. For some reason React wants a key value for each instance of a component <Island key={player.id} /> idk why, it was in the tutorial.